assets/src/components/vehiclePropertiesCard.tsx

```diff
@@ -0,0 +1,291 @@
+import React, { ReactNode, useContext, useId } from "react"
+import { useRoute } from "../contexts/routesContext"
+import { StateDispatchContext } from "../contexts/stateDispatchContext"
+import { className } from "../helpers/dom"
+import vehicleLabel from "../helpers/vehicleLabel"
+import { useCurrentTimeSeconds } from "../hooks/useCurrentTime"
+import { useNearestIntersection } from "../hooks/useNearestIntersection"
+import { emptyLadderDirectionsByRouteId } from "../models/ladderDirection"
+import { currentRouteTab } from "../models/routeTab"
+import { directionName } from "../models/vehicle"
+import {
+  drawnStatus,
+  humanReadableScheduleAdherence,
+  statusClasses,
+} from "../models/vehicleStatus"
+import { Vehicle } from "../realtime"
+import { secondsToMinutes } from "../util/dateTime"
+import { CloseButton2 } from "./closeButton"
+import { RouteVariantName2 } from "./routeVariantName"
+import StreetViewButton, { WorldPositionBearing } from "./streetViewButton"
+import { Size, VehicleIcon2, vehicleOrientation } from "./vehicleIcon"
+
+interface VehicleProp {
+  vehicle: Vehicle
+}
+
+//#region Card Title Bar
+const DataStaleTime = ({
+  timestamp,
+}: {
+  timestamp: number
+}): React.ReactElement => {
+  const epochNowInSeconds = useCurrentTimeSeconds()
+  return (
+    <output aria-label="Last Updated Time" className="data-stale-time">
+      Updated {epochNowInSeconds - timestamp} sec ago
+    </output>
+  )
+}
+
+const VehicleDataStaleTime = ({ vehicle }: VehicleProp): React.ReactElement => (
+  <DataStaleTime timestamp={vehicle.timestamp} />
+)
+//#endregion
+
+//#region Vehicle Summary
+
+const VehicleIcon1 = ({ vehicle }: VehicleProp): React.ReactElement => {
+  const [{ routeTabs, userSettings }] = useContext(StateDispatchContext)
+  const currentTab = currentRouteTab(routeTabs)
+  const ladderDirections = currentTab
+    ? currentTab.ladderDirections
+    : emptyLadderDirectionsByRouteId
+
+  return (
+    <div className="m-vehicle-route-summary__icon">
+      <VehicleIcon2
+        size={Size.Large}
+        orientation={vehicleOrientation(vehicle, ladderDirections)}
+        label={vehicleLabel(vehicle, userSettings)}
+        variant={vehicle.viaVariant}
+        status={drawnStatus(vehicle)}
+        userSettings={userSettings}
+      />
+    </div>
+  )
+}
+
+const VehicleRouteDirection = ({
+  vehicle,
+}: VehicleProp): React.ReactElement => {
+  const route = useRoute(vehicle.routeId)
+  return (
+    <output title="Route Direction" className="m-vehicle-route-direction">
+      {directionName(vehicle, route)}
+    </output>
+  )
+}
+
+const VisualSeparator = (): React.ReactElement => (
+  // Visual accent to provide separation between elements
+  // This object is strictly for visual presentation
+  <object
+    className="m-vehicle-route-summary__separator"
+    role="separator"
+    aria-hidden={true}
+    aria-label="presentation separator"
+  />
+)
```

Comment:
**praise:** I really like having this stuff separated out into smaller components, it definitely helps a bit with the readability of some otherwise-complex code.